### DISCUSSION TITLE
Compinit-like partial word matching

### DISCUSSION THREAD

**Discussion by NewtonChutney** [marlonrichert/zsh-autocomplete#595] [Q&A] [answered]
I'd like to have partial word completions, like expanding `f.b` to `foo.bar` for directories/file paths.

You get this config with:
-  `autoload - Uz zsh-newuser-install; zsh-newuser-install -f`
- Option 2 [use the new completion system] 
- Continue to the main menu [ `y`, `enter` x3 ] 
- Option 4 [for particular types of completion]
- Option 1 [for file completion]
- Option 5 [configure how multiple paths ate expanded and displayed]
- then complete the other config settings [`y`, `y`, `n`] 
- save your settings [ `0` x3, `y` x2, q] 

How can I have this behavior included by default, or can I fall back to such behavior after I press tab twice? 

I'm currently trying to open a directory 3 levels deep, with Helix editor, hx..
`hx d<tab>` expands properly to the first folder, `dev` but `hx d/p/r<tab>` doesn't expand..
It does expand to `dev/python/repo` with compinit and the mentioned config..

**Comment by marlonrichert** [comment, marked as answer]
Add to your `.zshrc` file _after_ sourcing Autocomplete:
```sh
zstyle ':completion:*:paths' path-completion yes
```
